src/ai/backend/common/events/dispatcher.py

```diff
@@ -646,4 +648,24 @@ async def produce_event(
             b"source": source_bytes,
             b"args": msgpack.packb(event.serialize()),
         }
+        # TODO: impl anycast message queue
```

Comment:
Probably `anycast` messages will use the current message queue implementation as is, and it seems like we need to create a new broadcast method.
Since the places that use broadcast messages are doing subscribe operations that can almost be volatile, we are considering changing it to a pubsub-based approach.